src/Storage/Services/RegisterService.cs

```diff
@@ -85,7 +85,7 @@ public async Task<Party> GetParty(int partyId)
         }
 
         /// <inheritdoc/>
-        public async Task<int> PartyLookup(string orgNo, string person)
+        public async Task<int> PartyLookup(string person, string orgNo)
```

Comment:
interface says orgNo is first input, not person. I would suggest Using the PartyLookup object as input to this method to avoid confusion with ordering of input parameters

Reply:
Correct! 👍